packages/mui-material/src/styles/createPalette.js

```diff
@@ -58,7 +58,7 @@ export const dark = {
   },
   divider: 'rgba(255, 255, 255, 0.12)',
   background: {
-    paper: '#121212',
+    paper: '#1A2027',
```

Comment:
@oliviertassinari After adjusting the demo background color, the paper bg color still looked off, so I felt the need of changing that. Letting you know because I don't know if the `#121212` was something from Material that we "couldn't" change but I'm confident that it's better now.